src/utils/url.utils.ts

```diff
@@ -104,3 +111,47 @@ export const getWebsocketURL = () => {
 
     return `wss://${serverURL}/websockets/v3?app_id=${getAppId()}&l=${language}&brand=${AppIDConstants.appBrand}`;
 };
+
+/**
+ * Extracts query parameters from the URL by parsing the current window's URL search parameters for the specified key.
+ * It returns the query parameters associated with the given key.
+ *
+ * @param {QueryParameters} key - The query parameter we want. (you can see all of them in the URLConstants.queryParameters)
+ * @returns {string | null} A string containing query parameter associated with the given key.
+ */
+export const getURLParameters = (key: QueryParameters) => {
```

Comment:
minor: name suggests that it accepts plural "parameters" while it does accept single key, 
so better name would be: "getURLParameter" or even better: "getQueryParameter" (as URL Parameter can be anyting including protocol, domain, path, port, anything, while query paramter clearly states what does that function actually do and is less confusing)

Reply:
You are right @wojciech-deriv will update this 🙏